src/core/distributions/phylogenetics/tree/birthdeath/AbstractBirthDeathProcess.cpp

```diff
@@ -126,10 +127,18 @@ std::vector<double> AbstractBirthDeathProcess::simulateDivergenceTimes(size_t n,
     for (size_t i = 0; i < n; ++i)
     {
         double t = 0.0;
-
-        do {
+        int ntries = 0;
+        
+        do { 
             t = simulateDivergenceTime(origin, present);
-        } while ( t < min );
+            ntries ++;
+        } while (t < min && ntries < 1000);
+
+        if(t < min) {
```

Comment:
Do you think this is safe or would it be better to throw an error?

Reply:
my thinking was that this is primarily used to initialize the run so it doesn't matter too much what the ages are as long as they work with the constraints...
if we just throw an error then we're back with the initial issue which is that some analyses cannot start because it can't find a valid tree